User/Library/UserPcd.c

```diff
@@ -50,4 +50,4 @@ BOOLEAN  _gPcd_FeatureFlag_PcdFatReadOnlyMode                        = _PCD_VALU
 UINT32   _gPcd_BinaryPatch_PcdSerialRegisterStride                   = 0;
 UINT8    _gPcd_FixedAtBuild_PcdUefiImageFormatSupportNonFv           = 0x00;
```

Comment:
Should this line be modified to match the default (0x02) which I believe is used unmodified in OC.